src/serverlist.cpp

```diff
@@ -205,9 +205,15 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
         // [this server country as QLocale ID]
         bool      ok;
         const int iCountry = slServInfoSeparateParams[2].toInt ( &ok );
-        if ( ok && iCountry >= 0 && iCountry <= QLocale::LastCountry )
+        if ( ok && iCountry >= 0 && CLocale::IsCountryCodeSupported ( iCountry ) )
```

Comment:
@pljones This is the place where we had a sanity check (`<= QLocale::LastCountry`) already. I'm removing it here in favor of the split Qt5/Qt6 check in `src/util.cpp`.

Reply:
Wouldn't it be better to pass `iCountry` to `CLocale::WireFormatCountryCodeToQtCountry ( iCountry )` and see whether that complains?  Doesn't `CLocale::IsCountryCodeSupported ( iCountry )` potentially allow a Qt6 code (on a Qt6 build into the `if`?

You're also allowed to throw a `CGenErr` in the constructor if something isn't valid.  So if the user tries a Qt6 code here (whilst it's still "wire format"), we can tell them.

It might be nice to `qInfo` out the selected country so that we log what we actually used (the _name_, not the number) -- where the numbers aren't the same between Qt5 and Qt6, it would help the user spot potential problems.

Reply:
Might be a nice "side effect" of #2845 to include this.